estela-api/core/models.py

```diff
@@ -431,6 +439,15 @@ class UsageRecord(models.Model):
     items_data_size = models.PositiveBigIntegerField(
         help_text="Amount in bytes occupied by items in the database"
     )
+    residential_proxy_usage = models.PositiveBigIntegerField(
+        default=0,
+        help_text="Amount in bytes occupied by residential proxy responses in the database",
+    )
+    datacenter_proxy_usage = models.PositiveBigIntegerField(
+        default=0,
+        help_text="Amount in bytes occupied by datacenter proxy responses in the database",
+    )
```

Comment:
We need to give this a bit more thought. What if we want to add ISP or mobile proxies? We would also need to add a field for each. I am not sure if there are more kinds of proxies, but it isn't a good design having to add more fields for each of them here 🤔